full-stack-tests/imodelhub-client/src/TestIModelHubOidcAuthorizationClient.ts

```diff
@@ -4,10 +4,10 @@
 *--------------------------------------------------------------------------------------------*/
 
 import { AccessToken, BeEvent } from "@itwin/core-bentley";
-import { FrontendAuthorizationClient } from "@bentley/frontend-authorization-client";
 import { TestUtility } from "@itwin/oidc-signin-tool";
+import { AuthorizationClient } from "@bentley/itwin-client";

```

Comment:
Can we move the interface of "AuthorizationClient" to `core-common` so it can be a shared interface without having to take a dependency on itwin-client?

I'm removing itwin-client entirely in my other PR.

Reply:
Just moved it. It is now in `core-common` and references have been changed.